plugins/woocommerce/client/blocks/assets/js/atomic/blocks/product-elements/rating/block.json

```diff
@@ -26,18 +26,9 @@
 			"default": false
 		}
 	},
-	"usesContext": [
-		"query",
-		"queryId",
-		"postId"
-	],
-	"keywords": [
-		"WooCommerce"
-	],
-	"supports": {
-		"align": true
```

Comment:
What's the reason for removing `"align": true`?

Reply:
It's overridden by support.ts values and essentially ignored when registering [here](https://github.com/woocommerce/woocommerce/blob/542f108d15ce5ad06e0a343bc76f6ee94de09c76/plugins/woocommerce/client/blocks/assets/js/atomic/blocks/product-elements/rating/index.ts#L20).

That's the case everywhere where I removed it.